agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -1668,6 +1699,7 @@ func TestHandleUpdateService(t *testing.T) {
 					},
 				},
 			},
+			expectedImportedServicesCount: 0, // IRL, this count would be 1, but we pre-seed the redis service
```

Comment:
In the future, one way you could have worked around this would be to seed the imported service count too.

Reply:
I had a version of this where there was a seeding of imported services count too so I'm brining that back!